apps/ocp-plugin/deploy/helm/flightctl-ocp-ui/templates/flightctl-ui-network-policy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Shouldn't we consider the "standalone" deployment the base one, and the OCP one the overwrites?

Eg. if we had a different plugin, it would extend the OCP one which feels a bit weird.

What I mean is to define the NP for the standalone deployments, then have the OCP plugin import it.